bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestAppProfile.java

```diff
@@ -61,27 +63,36 @@
 public class TestAppProfile {
   private static final String TABLE_ID = "fake-table";
 
-  private FakeDataService fakeDataService;
-  private Server server;
+  private static FakeDataService fakeDataService = new FakeDataService();
```

Comment:
Why static?

Reply:
To instantiate Server only once for all the tests. currently, Server creation is in setUp().

Reply:
Server creation is cheap, in general we should keep tests isolated as much as possible unless there is a large penalty. But as discussed below, I'm going to close this for the time being and we will come back to it in the future